src/_includes/default-layout.njk

```diff
@@ -44,20 +44,36 @@
 
 <body class="{{title}}">
 
-    <header class="header">
+    <header id="header" class="header">
         <a href="/" aria-label="Home Page" class="logo img-wrap" style="--aspect-ratio:440/80;">
             {% include "assets/logo/vermont-code-camp-logo-light.svg" %}
             {# <img src="/assets/vermont-code-camp-logo-light.svg" alt="Vermont Code Camp" /> #}
         </a>
     </header>
-    <nav class="header">
-        <ul class="nav-buttons">
-            <li><a href="/" class='code {{ "active" if "/" == page.url }}'>Attend<span class="hide-xs text-muted" aria-hidden="true">();</span></a></li>
-            <li><a href="/speak/" class='code {{ "active" if "/speak/" in page.url }}'>Speak<span class="hide-xs text-muted" aria-hidden="true">();</span></a></li>
-            <li><a href="/sponsor/" class='code {{ "active" if "/sponsor/" in page.url }}'>Sponsor<span class="hide-xs text-muted" aria-hidden="true">();</span></a></li>
-            <li><a href="/conduct/" class='code {{ "active" if "/conduct/" in page.url }}'>Conduct<span class="hide-xs text-muted" aria-hidden="true">();</span></a></li>
+    
+    <nav id="headerNav" class="header">
+        <ul id="navButtons" class="nav-buttons">
+            <li><a href="/" class='code home {{ "active" if "/" == page.url }}'>Home</a></li>
+            <li><a href="/schedule/" class='code {{ "active" if "/schedule/" in page.url }}'>Schedule</a></li>
+            <li><a href="/sessions/" class='code {{ "active" if "/sessions/" in page.url }}'>Sessions</a></li>
+            <li><a href="/speakers/" class='code {{ "active" if "/speakers/" in page.url }}'>Speakers</a></li>
+            <li><a href="/parking/" class='code {{ "active" if "/parking/" in page.url }}'>Parking</a></li>
+            <li><a href="/conduct/" class='code {{ "active" if "/conduct/" in page.url }}'>Conduct</a></li>
+            <li><a href="/diversity/" class='code {{ "active" if "/diversity/" in page.url }}'>Diversity</a></li>
+            <li><a href="/sponsor/" class='code {{ "active" if "/sponsor/" in page.url }}'>Sponsor</a></li>
         </ul>
+        <a href="#footer-nav" id="navToggle" class='nav-toggle'> </a>
```

Comment:
no reason for `footer-nav` here, right?  Just a copy/ paste thing I'm guessing.  I'll push up some changes, but just wanted to make sure I wasn't missing anything

Reply:
Nope, that's a fallback for if JS is turned off or fails!  Scrolls the viewport to the footer where all the links are available.